bigbone/src/main/kotlin/social/bigbone/MastodonClient.kt

```diff
@@ -417,11 +432,32 @@ private constructor(
         private val gson = Gson()
         private var accessToken: String? = null
         private var debug = false
+        private var scheme = "https"
+        private var port = 443
+        private var trustAllCerts = false
 
         fun accessToken(accessToken: String) = apply {
             this.accessToken = accessToken
         }
 
+        fun withHttpsDisabled() = apply {
```

Comment:
Can public Mastodon servers run on an HTTP scheme? If not, we should at least advise against using this in production, similar to the documentation of `withTrustAllCerts()`.

Reply:
For production setups this should clearly not be used. Back then, I have added it for testing-purposes only. My plan was to run local Mastodon instances on HTTP only (for simplicity reasons). But I've soon realized, that Ruby on Rails together with Mastodon code works best using HTTPS (saw a comment somewhere, people claiming that not everything might work as expected, if stuff is running on HTTP). So I've built the test setup based on HTTPS with certificates. 

Now, as the HTTPS setup is working, we could simply kill this method.